src/modules/new-request-form/NewRequestForm.tsx

```diff
@@ -3,12 +3,26 @@ import { TextInput } from "./fields/TextInput";
 import { TextArea } from "./fields/TextArea";
 import { DropDown } from "./fields/DropDown";
 import { TicketFormField } from "./ticket-form-field/TicketFormField";
+import type { FormEventHandler } from "react";
+import { Button } from "@zendeskgarden/react-buttons";
+import styled from "styled-components";
+import { Alert } from "@zendeskgarden/react-notifications";
 
 export interface NewRequestFormProps {
   ticketForms: TicketForm[];
   requestForm: RequestForm;
 }
 
+const StyledForm = styled.form`
+  display: flex;
+  flex-direction: column;
+  gap: ${(props) => props.theme.space.md};
+`;
+
+const Footer = styled.div`
```

Comment:
Nit: Should we keep consistency and call this `StyledFooter`?
Or maybe simpler to to just have `Form` and `Footer`.